### PR TITLE
added Elesticsearch resource

### PR DESCRIPTION
**Pull request by avivgu**

### CODE REVIEW

File: terraform/es.tf [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  ebs_options {
+    ebs_enabled = true
+    volume_size = 50
+  }
+
+}
+
+resource "aws_elasticsearch_domain_policy" "monitoring-framework-policy" {
+  domain_name = aws_elasticsearch_domain.monitoring-framework.domain_name
+  access_policies = <<POLICIES
```
Comment (nimrodkor) [inline terraform/es.tf:22]: @avivgu Please change to aws_iam_policy_document

File: terraform/es.tf [hunk 2]
```diff
@@ -0,0 +1,35 @@
+resource "aws_elasticsearch_domain" "monitoring-framework" {
+  domain_name           = "tg-${var.environment}-es"
+  elasticsearch_version = "2.3"
+
+  cluster_config {
+    instance_type            = "m4.large.elasticsearch"
```
Comment (nimrodkor) [inline terraform/es.tf:6]: @elaygl Please change to `t2.small.elasticsearch`. our AWS bill will be happier that way

File: terraform/es.tf (hunk 1 above)
Comment (avivgu) [reply terraform/es.tf]: done

File: terraform/es.tf (hunk 2 above)
Comment (avivgu) [reply terraform/es.tf]: done

**Pull request merged**